internal/pkg/mojang/profile.go

```diff
@@ -82,7 +82,7 @@ func (c *Client) ChangeSkin(ctx context.Context, accountToken string, texture st
 }
 
 func (c *Client) ChangeCape(ctx context.Context, accountToken string, cape string) (*ProfileInformationResponse, error) {
-	endpoint := "/capes/active"
+	endpoint := "capes/active"
```

Comment:
nitpick: other places (modrinth client at least) leave the slashes to the final call, should keep standard